program/src/geometric.cpp

```diff
@@ -8,7 +8,7 @@ bool Geometric::is_a(vector<int> given) {
     int first = given[0];
     int second = given[1];
 
-    int q = second / first;
+    float q = second / float(first);
```

Comment:
a przedtem zastanawiałem sie czemu tutaj jest `float(first)` skoro i tak mam typ `int` XD to usunąłem `float(*)`

Reply:
XD